examples/3d/light_textures.rs

```diff
@@ -155,12 +155,6 @@ fn setup(
     mut meshes: ResMut<Assets<Mesh>>,
     mut materials: ResMut<Assets<StandardMaterial>>,
 ) {
-    // Error out if the light textures feature isn't enabled
-    if !cfg!(feature = "pbr_light_textures") {
-        eprintln!("Bevy was compiled without light texture support. Run with `--features=pbr_light_textures` to enable.");
-        process::exit(1);
```

Comment:
Can remove the import for process?

Reply:
it was also used for another thing, replaced it with using `AppExit`